ui/src/overridden/frontsite/Document/DocumentDetails/DocumentCirculation/LiteratureKeyword.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export const LiteratureKeyword = ({ keywords }) => {
+  const keywordsIsValidArray = Array.isArray(keywords);
+
+  if (!keywordsIsValidArray) return null;
```

Comment:
In which case the keywords are not an array?

Reply:
Sorry, my question was more to challenge if keywords should be empty! Instead of checking here, I would probably ensure that the parent component is always providing an array! That way the component remains simple.

Reply:
ah i see! Well it is an overridden component that receives more general data so that it can be used flexibly. I would say the component is pretty simple either way :sweat_smile: